e2e-tests/js_scripts/0008-update-runtime-multisig-scheduler.js

```diff
@@ -0,0 +1,153 @@
+    const BOB   = '5FHneW46xGXgs5mUiveU4sbTyGBzmstUspZC92UhjJM694ty';
+    const CHARLIE = '5FLSigC9HGRKVhB9FiEo4Y3koPsNmBmLJbpXg2mp1hXcS59Y';
+
+    // Use the keyring to generate accounts
+    const keyring = new zombie.Keyring({ type: 'sr25519' });
+    const alice = keyring.addFromUri('//Alice');
+    const bob = keyring.addFromUri('//Bob');
+    const charlie = keyring.addFromUri('//Charlie');
+    const ACME = 1_000_000_000_000_000_000n;
+
+    /*****************************************************************************************************
+     *************************************** CREATE MULTISIG ACCOUNT *************************************
+     *****************************************************************************************************/
+    const threshold = 2;
+    const multisigAddress = util.createKeyMulti([alice.address, bob.address, charlie.address], threshold);
+    const SS58Prefix = 42;
+    const Ss58MultiAddress = util.encodeAddress(multisigAddress, SS58Prefix);
+    console.log(`multisigAddress ${Ss58MultiAddress}`);
+
+
+    /*****************************************************************************************************
+     *************************************** SET MULTISIG AS SUDO ACCOUNT ********************************
+     *****************************************************************************************************/
+
+    const newSudoKey = api.tx.sudo.setKey(Ss58MultiAddress);
+    await submitExtrinsic(newSudoKey, alice, BlockUntil.Finalized, undefined);
+
+    const newSudoKeyOption = await api.query.sudo.key();
+    if (newSudoKeyOption.isSome && newSudoKeyOption.unwrap().toString() !== Ss58MultiAddress) {
+        return ReturnCode.ErrWrongSudoAccount;
+    }
+
+    /*****************************************************************************************************
+     *************************************** GET CURRENT RUNTIME VERSION******* **************************
+     *****************************************************************************************************/
+
+    const currentRuntimeVersion = (await api.rpc.state.getRuntimeVersion()).specVersion.toNumber();
+
+    /*****************************************************************************************************
+     *************************************** SCHEDULE RUNTIME UPGRADE*************************************
+     *****************************************************************************************************/
+
+    // Retrieve the runtime to upgrade
+    const sudoAccount = await api.query.sudo.key()
+    const code = fs.readFileSync('./new-runtime/nh_runtime.wasm').toString('hex');
+    const updateRuntimeCall = api.tx.system && api.tx.system.setCode
+        ? api.tx.system.setCode(`0x${code}`) // For newer versions of Substrate
+        : api.tx.consensus.setCode(`0x${code}`); // For previous versions
+    const sudoUpdateRuntimeCall = api.tx.sudo.sudo(updateRuntimeCall)
+
```

Comment:
You did it the wrong order: I think that the right one is `Multisig[Sudo[Schedule[SetCode]]]` in this order you don't need of `EnsureRootOrSudo` to update code via multisig.
`Sudo` ensure that you call `Schedule` as `Root` and `Schedule` execute scheduled action aways as `Root`.